lib/hooks/command-not-found/suggest.ts

```diff
@@ -44,16 +46,36 @@ const hook: Hook<'command-not-found'> = async function (
 		return _.minBy(commandIDs, (c) => Levenshtein.get(cmd, c))!;
 	}
 
+	const suggestions: string[] = [];
+	suggestions.push(closest(commandId).replace(':', ' ') || '');
+
+	// opts.argv contains everything after the first command word
+	// if there's something there, also test if it might be a double
+	// word command spelt wrongly, rather than command args.
+	if (opts.argv?.[0]) {
+		suggestions.unshift(
+			closest(`${commandId}: + ${opts.argv[0]}`).replace(':', ' ') || '',
+		);
+	}
+
+	// Output suggestions
 	console.error(
```

Comment:
Here we're using `console.error`, but below using `console.log`. One could reason that only "proper errors" (as opposed to "info messages") should go to `stderr`, but my preferred reasoning :-) is that any error/warn/info messages should go to stderr (see other comment). But feel free to disagree! :-)